### PR TITLE
[Doc] Add more examples of adding attributes in Live Components

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| Issues        | -
| License       | MIT

Note: I tried this in a Symfony app and it didn't work ... so I'm opening this PR to ask you if this should work or if `addAttribute()` is only for empty attributes that don't define a value.

Thanks!

### CODE REVIEW

**Comment by smnandre** [comment]
It only works with attributes in a toggle way .. you may want to use addClass there i think

**Comment by javiereguiluz** [comment]
Thanks for the review. I updated the PR to show an example that won't work and explain that explicitly. Thanks.

File: src/LiveComponent/doc/index.rst [hunk 1]
```diff
@@ -931,14 +931,18 @@ a class:
     <!-- add multiple classes when loading -->
     <div data-loading="addClass(opacity-50 disabled)">...</div>
```
Comment (smnandre) [inline src/LiveComponent/doc/index.rst]: Realizing after your modification we should not use "disabled" both in addClass and addAttribute, it does not help.. Could 
```suggestion
    <div data-loading="addClass(opacity-50 text-warning)">...</div>
```

**Review by smnandre — APPROVED** [review]

File: src/LiveComponent/doc/index.rst [hunk 2]
```diff
@@ -931,14 +931,18 @@ a class:
...
-Sometimes you may want to add or remove an attribute when loading. That
+Sometimes you may want to add or remove an HTML attribute when loading. That
 can be accomplished with ``addAttribute`` or ``removeAttribute``:
 
 .. code-block:: html+twig
 
     <!-- add the "disabled" attribute when loading -->
     <div data-loading="addAttribute(disabled)">...</div>
 
+    <!-- this feature only works for empty attributes that don't define any value;
```
Comment (kbond) [inline src/LiveComponent/doc/index.rst]: What about wrapping this in a caution annotation? I'm worried readers might skip over _the following example won't work_ and think it's valid

Comment (weaverryan) [reply src/LiveComponent/doc/index.rst]: I agree - excellent detail to point out

**Review by weaverryan — CHANGES_REQUESTED** [review]

**Comment by javiereguiluz** [comment]
Sorry I forgot about this PR. I did all the changes that you suggested. Thanks for the reviews!

**Comment by kbond** [comment]
> I did all the changes that you suggested.

@javiereguiluz, did you miss pushing these changes?

**Comment by javiereguiluz** [comment]
@kbond yes, sorry 😊

**Review by kbond — APPROVED** [review]

File: src/LiveComponent/doc/index.rst (hunk 1 above)
Comment (smnandre) [reply src/LiveComponent/doc/index.rst]: Missed a bit of message there, thank you @javiereguiluz for guessing the end of my sentence and fixing it ❤️

**Comment by weaverryan** [comment]
Thanks Javier!

**Pull request merged**